tox.ini

```diff
@@ -6,15 +6,14 @@ envlist =
 [testenv]
 setenv =
     PYTHONDONTWRITEBYTECODE=1
-install_command = pip install --no-deps {opts} {packages}
-deps = -rrequirements.txt
-commands = pytest -p no:randomly {posargs}
+deps = pipenv
+commands =
+    pipenv install --dev --system --deploy --ignore-pipfile
+    pipenv run pytest -p no:randomly {posargs}
```

Comment:
afaiu `tox` is now creating virtualenvs with just `pipenv` installed, under which `pipenv` is then creating *another* virtualenv with the dependencies? Perhaps this plugin is a better way to do it? https://pypi.org/project/tox-pipenv/

Reply:
It does not create a virtualenv due to:

```
Courtesy Notice: Pipenv found itself running within a virtual environment, so it will automatically use that environment, instead of creating its own for any project. You can set PIPENV_IGNORE_VIRTUALENVS=1 to force pipenv to ignore that environment and create its own instead.
```

Maybe I need to remove `--system` flag. Still not sure about it.

Reply:
Considering `tox-pipenv`: it does not look mature enough. And I am not sure how it works.

But we definitely can try.